CarRentals/CarRentals/Controllers/CarCRUD.cs

```diff
@@ -57,7 +53,6 @@ public Car Get(int id)
             return Result;
         }
 
-
         public Car Update(Car car)
```

Comment:
I see here you implemented a `for` statement and in the Get you used a `foreach`, in both you never stop iterating after finding the element meaning if the element you are looking for is the first of a sequence of 1000 elements, you would still iterate through all of them before returning the result. 
also, a suggestion: feel free to investigate about LINQ (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/introduction-to-linq-queries) and see if you can implement it here